bankid-idp/bankid-idp-backend/src/main/java/se/swedenconnect/bankid/idp/authn/service/BankIdService.java

```diff
@@ -167,10 +167,9 @@ private Mono<ApiResponse> handleError(final Throwable e, final PollRequest reque
    * @param request the {@link PollRequest}
    * @return a {@link BankIdSessionData}
    */
-  private Mono<BankIdSessionData> reInitIfExpired(final PollRequest request) {
+  private Mono<BankIdSessionData> reInitIfExpired(final PollRequest request, BankIdSessionData bankIdSessionData) {
```

Comment:
Minor thing, but. Since the other parameter is final, the bankIdSessionData should also be marked as final